app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  skip_before_action :require_login, only: [:new, :create]
+
```

Comment:
This is a really great idea to restrict what users can or can't do if they are logged in. I am not able to find the require login method defined in the application controller, so I am not sure if this is working as expected.